packages/proxy/src/handler.ts

```diff
@@ -66,40 +63,73 @@ function isRedirect(
 
 export async function handler(event: CloudFrontRequestEvent) {
   const { request } = event.Records[0].cf;
+  const domainName = request.origin!.s3!.customHeaders['x-env-domain-name'][0].value;
   const configEndpoint = request.origin!.s3!.customHeaders[
     'x-env-config-endpoint'
   ][0].value;
-  const apiEndpoint = request.origin!.s3!.customHeaders['x-env-api-endpoint'][0]
+  let apiEndpoint = request.origin!.s3!.customHeaders['x-env-api-endpoint'][0]
     .value;
+
   let headers: Record<string, string> = {};
```

Comment:
Don't think this variable is mutable is it?
```suggestion
  const headers: Record<string, string> = {};
```

Reply:
It's being re-assigned in 211.